dockerfiles/image.sh

```diff
@@ -277,14 +302,18 @@ incr() {
 }
 
 install() {
-    if [ ! -e $DEB_FILE ]; then
+    if [ -d $DEB_FILE ]; then
```

Comment:
the usage says:
```
    echo "  install <file> install from a local deb file"
```

but I see it's interpreting as a dir (conditionally), probably just needs the usage to be clarified.

An example of new usage in JIRA description/notes would be good too.